Bixal/move.mil
# Feature/parse zip3 file

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Address pivotal ticket #156844285
- Create a DC command called `parse` to parse move.mil files and generate normalized json files for
zip3. For the PPM tool

## Testing

To verify the changes proposed in this pull request…

1. Pull branch changes, do a `composer install`, `make cim`.
1. Since there was a Drupal core update, execute `docker-compose run php drupal updb` to fix the external links table (it'll be broken otherwise).
1. Execute `docker-compose run php drupal parser` choose zip3.
1. Check that the json file is generated at `web/sites/default/files/tools/data`.
